PROCESSES.md

```diff
@@ -2,10 +2,100 @@
 
 This document outlines processes and procedures for some common tasks in the charts repository.
 
+## Review Process
+
+One of the Chart maintainers should review the PR.
+If everything is fine then the PR should be [approved](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/approving-a-pull-request-with-required-reviews).
+The one who approves the PR should also merge it directly.
+In case the reviewer wants someone else to have a look on it it should be mentioned as a comment so that it's transparent for everyone.
+
+As a chart maintainer can not approve it's own PRs every chart should have at least two maintainers.
+For charts where this is not the case or where none of the other maintainers does a review within two weeks the maintainer who created the PR could request a review from a repository admin instead.
+
 ## Adding chart maintainers
 
 Chart maintainers are defined within the chart itself.
 So the procedure for adding maintainers is to add them there.
-The pull request which does that should also update [CODEOWNERS](./github/CODEOWNERS) file to that the new maintainer is able to approve pull requests.
+The pull request which does that should also update [CODEOWNERS](./.github/CODEOWNERS) file to that the new maintainer is able to approve pull requests.
 One of the existing chart maintainers needs to approve the PR in addition one of the repository admins needs to approve it.
 They are then responsible for also granting the new maintainer write permissions to this repository.
+
+## GitHub Settings
+
+As not everyone is able to see which settings are configured for this repository these are also documented here.
+Changing settings outlined should only be done once a PR is approved which documents those changes.
+
+### Merge Settings
+
+Only squash merge is allowed in this repository:
+
+> Allow squash merging
+> Combine all commits from the head branch into a single commit in the base branch.
+
+"Allow merge commits" and "Allow rebase merging" are disabled to keep the history simple and clean.
+
+### Repository Access
+
+Repository access and permissions are managed via the GitHub teams.
+
+| GitHub Team | Repository Access |
+| ----------- | ---- |
+| [helm-charts-maintainers](https://github.com/orgs/prometheus-community/teams/helm-charts-maintainers) | Write |
+| [helm-charts-admins](https://github.com/orgs/prometheus-community/teams/helm-charts-admins) | Admin |
+
+Chart maintainers are members of [helm-charts-maintainers](https://github.com/orgs/prometheus-community/teams/helm-charts-maintainers).
```

Comment:
We can't link to teams, but we can adopt the syntax 😄 see https://github.com/github/markup/issues/209
```suggestion
Chart maintainers are members of [@prometheus-community/helm-charts-maintainers](https://github.com/orgs/prometheus-community/teams/helm-charts-maintainers).
```